assets/general-settings/actions.js

```diff
@@ -0,0 +1,25 @@
+import server from 'server';
+import { gettext, notify } from 'utils';
+
+export const UPDATE_VALUES = 'UPDATE_VALUES';
+function updateValues(values) {
+    return {type: UPDATE_VALUES, values: values};
+}
+
+export function save(values) {
+    return (dispatch) => {
+        server.post('/settings/general_settings', values)
+            .then(() => {
+                notify.success(gettext('Settings were updated successfully.'));
+                dispatch(updateValues(values));
+            }, (reason) => {
+                console.error('error', reason);
+                notify.error(gettext('Ups, there was an error when updating settings.'));
```

Comment:
I'd rather say `Oops` here :)

Reply:
right, there must be a trademark for it :wink: